tests/UnifiedSpecTests/UnifiedSpecTest.php

```diff
@@ -187,6 +187,8 @@ class UnifiedSpecTest extends FunctionalTestCase
         'crud/db-aggregate-write-readPreference: Database-level aggregate with $out omits read preference for pre-5.0 server' => 'PHPLIB-1458',
         'crud/db-aggregate-write-readPreference: Database-level aggregate with $merge includes read preference for 5.0+ server' => 'PHPLIB-1458',
         'crud/db-aggregate-write-readPreference: Database-level aggregate with $merge omits read preference for pre-5.0 server' => 'PHPLIB-1458',
+        // GridFS deprecated fields are removed
+        'gridfs/gridfs-upload: upload when contentType is provided' => 'Deprecated fields are removed',
```

Comment:
Did you also mean to disable the tests in `gridfs-upload-disableMD5.json`?

Also, can `disableMD5` and `contentType` be removed from the `uploadWithId` and `upload` arrays in `MongoDB\Tests\UnifiedSpecTests\Util::$args`? If you're skipping all tests including those deprecated options, I expect so.

Reply:
It wasn't clear to me. There's no need for these tests, which now duplicate others.